### PR TITLE
DROID-2663 Version history | Object version preview

### PR DESCRIPTION
**Pull request by konstantiniiv**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt [hunk 1]
```diff
@@ -2003,10 +2005,6 @@ open class EditorFragment : NavigationFragment<FragmentEditorBinding>(R.layout.f
         vm.onSelectProgrammingLanguageClicked(target, key)
     }
 
-    override fun onMoveToBinSuccess() {
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:2006]: why has it been deleted?

File: app/src/main/java/com/anytypeio/anytype/ui/editor/sheets/ObjectMenuBaseFragment.kt [hunk 2]
```diff
@@ -354,7 +354,6 @@ abstract class ObjectMenuBaseFragment :
     }
 
     interface DocumentMenuActionReceiver {
-        fun onMoveToBinSuccess()
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/editor/sheets/ObjectMenuBaseFragment.kt:357]: idem

File: app/src/main/java/com/anytypeio/anytype/ui/history/VersionHistoryFragment.kt [hunk 3]
```diff
@@ -58,19 +141,19 @@ class VersionHistoryFragment : BaseBottomSheetComposeFragment() {
...
     }
 
     override fun releaseDependencies() {
-        componentManager().versionHistoryComponent.release(ctx + spaceId)
+        componentManager().versionHistoryComponent.release()
     }
 
     companion object {
+
+        const val DEFAULT_PADDING_TOP = 74
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/history/VersionHistoryFragment.kt:155]: `PADDING_TOP`

**Review by uburoiubu — APPROVED** [review]

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt (hunk 1 above)
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:2006]: we don't need it anymore, navigation is used in that case

File: app/src/main/java/com/anytypeio/anytype/ui/editor/sheets/ObjectMenuBaseFragment.kt (hunk 2 above)
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/ui/editor/sheets/ObjectMenuBaseFragment.kt:357]: we don't need it anymore, navigation is used in that case

File: presentation/src/main/java/com/anytypeio/anytype/presentation/editor/EditorViewModel.kt [hunk 4]
```diff
@@ -1221,10 +1221,6 @@ class EditorViewModel(
         exitBack()
     }
 
-    fun onMovedToBin() {
-        navigate(EventWrapper(AppNavigation.Command.Exit))
```
[the remark this answers is not in the document]
Comment (konstantiniiv) [reply presentation/src/main/java/com/anytypeio/anytype/presentation/editor/EditorViewModel.kt:1227]: same as I mentioned before

**Pull request merged**